docs/projects/human-menu.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: The Human Menu
+nav_order: 3
+parent: Projects
+---
+
+# The Human Menu
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## What is it?
+
+The [Human Menu](https://www.github.com/good-labs/human-menu/) project 
+provides a prototype for a "human menu," or a selection of items
+that contrasts the tendency to mindlessly browse social media or similar
+applications. The project is inspired by the problems outlined 
+by [The Center for Humane Technology](https://humanetech.com/problem/),
+specifically that our attentions are captured by addictive "digital slot 
+machines" to the detriment of our mental health.
+
+## Why do we need it?
+
+Humans use lists and suggestions to reduce cognitive load. Instead of
+offering a list of YouTube videos, endlessly scrolling social media posts,
+or phone apps, the Human Menu provides a selection of real world "human" options
+such as "Call your Mom," or "Do a puzzle."
+
+## Project Plan
+
+The web interface will present a simple list of choices (5-7 that are randomly selected)
+populated from an open source listing on GitHub that others can contribute to.
```

Comment:
A random list sounds like a good starting point but it would be great to have an option for a user to specify their own list. I'm thinking of a loved one who lost her mother in recent years and would probably get quite emotional if the Human Menu told her to call her mom. 😢

Reply:
Having GitHub posting means that someone could fork the entire repo and roll their own list. They could also contribute items to the one served here.